packages/strapi-design-system/src/Field/FieldLabel.js

```diff
@@ -17,19 +17,11 @@ const Action = styled(Flex)`
   }
 `;
 
-export const FieldLabel = ({ children, required, action, ...props }) => {
-  const { id } = useField();
+export const FieldLabel = ({ children, action, ...props }) => {
+  const { id, required } = useField();
```

Comment:
I think we should keep the required prop, because otherwise we'd introduce a breaking change. What do you think about:

```suggestion
export const FieldLabel = ({ children, action, required: requiredDeprecatedProp, ...props }) => {
  const { id, required: requiredField } = useField();
  const required = requiredDeprecatedProp || requiredField;
```

Reply:
that's a good catch, thank you!